src/encoded/types/analyte.py

```diff
@@ -0,0 +1,46 @@
+from typing import Iterable, List, Optional, Union
+
+from pyramid.request import Request
+from snovault import calculated_property, collection
+
+from .base import SMAHTItem, load_smaht_schema
+from .utils import CENTER_SUBMITTER_IDS_SCHEMA, get_unique_center_submitter_ids
+
+
+@collection(
+    name="analytes",
+    unique_key="accession",
+    properties={"title": "Analytes", "description": "Listing of analytes"},
+)
+class Analyte(SMAHTItem):
+    item_type = "analyte"
```

Comment:
Please docstring all new types so we have some basic info in code on what they are/what important relationships exist ie revlinks especially.